packages/table/src/tableBody.tsx

```diff
@@ -116,15 +141,24 @@ export class TableBody extends React.Component<ITableBodyProps, {}> {
             selectedRegionTransform,
         } = this.props;
 
-        const style = grid.getRect().sizeStyle();
-        const cells: Array<React.ReactElement<any>> = [];
+        // render cells in batches
+        this.batcher.startNewBatch();
         for (let rowIndex = rowIndexStart; rowIndex <= rowIndexEnd; rowIndex++) {
             for (let columnIndex = columnIndexStart; columnIndex <= columnIndexEnd; columnIndex++) {
-                const extremaClasses = grid.getExtremaClasses(rowIndex, columnIndex, rowIndexEnd, columnIndexEnd);
-                const isGhost = grid.isGhostIndex(rowIndex, columnIndex);
-                cells.push(this.renderCell(rowIndex, columnIndex, extremaClasses, isGhost));
+                this.batcher.addArgsToBatch(rowIndex, columnIndex);
             }
         }
+        this.batcher.removeOldAddNew((row: number, col: number) => {
+            const extremaClasses = grid.getExtremaClasses(row, col, rowIndexEnd, columnIndexEnd);
+            const isGhost = grid.isGhostIndex(row, col);
+            return this.renderCell(row, col, extremaClasses, isGhost);
+        });
+        if (!this.batcher.isDone()) {
+            this.batcher.idleCallback(() => this.forceUpdate());
```

Comment:
To avoid instantiating a new anonymous function on each `render`, can we just do `this.batcher.idleCallback(this.forceUpdate)`?